jobserver/views/users.py

```diff
@@ -48,7 +48,7 @@ def form_valid(self, form):
         user = User.objects.filter(email=form.cleaned_data["email"]).first()
 
         if user:
-            if not has_role(user, InteractiveReporter):
+            if InteractiveReporter not in user.all_roles:
```

Comment:
I know this is the existing behaviour, but are we potentially revealing the existence of a user account by returning back this message? Would it be better just to return back the message on line 80 regardless, but only email them the login link if they have the Interactive role?